atd-vzv/public/vz_logo.svg

```diff
@@ -0,0 +1,181 @@
+		c0,0.85,0.77,1.34,1.71,1.34c0.77,0,1.53-0.26,1.92-0.79V53.24z"/>
+	<path class="st0" d="M232.11,49.54h-1.51v-2.09h1.51v-0.49c0-1.98,1.28-3.17,3.07-3.17c0.68,0,1.3,0.15,1.73,0.38l-0.49,1.7
+		c-0.21-0.13-0.49-0.19-0.77-0.19c-0.68,0-1.13,0.45-1.13,1.32v0.45h1.85v2.09h-1.85v7.01h-2.41V49.54z M238.28,49.54h-1.51v-2.09
+		h1.51v-0.49c0-1.98,1.28-3.17,3.07-3.17c0.68,0,1.3,0.15,1.73,0.38l-0.49,1.73c-0.19-0.13-0.45-0.23-0.77-0.23
+		c-0.68,0-1.13,0.45-1.13,1.28v0.49h1.85v2.09h-1.85v7.01h-2.41V49.54z M243.67,44.85c0-0.79,0.66-1.43,1.43-1.43
+		c0.79,0,1.43,0.64,1.43,1.43c0,0.79-0.64,1.43-1.43,1.43C244.33,46.28,243.67,45.64,243.67,44.85 M243.91,47.45h2.39v9.1h-2.39
+		V47.45z"/>
+	<path class="st0" d="M252.92,47.22c1.87,0,3,0.81,3.6,1.64l-1.56,1.47c-0.43-0.64-1.09-0.98-1.92-0.98c-1.45,0-2.47,1.06-2.47,2.64
+		s1.02,2.66,2.47,2.66c0.83,0,1.49-0.38,1.92-0.98l1.56,1.45c-0.6,0.85-1.73,1.66-3.6,1.66c-2.81,0-4.82-1.98-4.82-4.79
+		C248.1,49.2,250.11,47.22,252.92,47.22"/>
+	<path class="st0" d="M269.17,55.38c-0.72,0.9-1.72,1.39-2.81,1.39c-2.3,0-4.03-1.73-4.03-4.77c0-2.97,1.72-4.79,4.03-4.79
+		c1.08,0,2.09,0.47,2.81,1.39v-4.64h2.41v12.57h-2.41V55.38z M269.17,50.39c-0.4-0.6-1.26-1.04-2.07-1.04c-1.36,0-2.3,1.07-2.3,2.66
+		c0,1.57,0.94,2.64,2.3,2.64c0.81,0,1.68-0.43,2.07-1.04V50.39z"/>
+	<path class="st0" d="M278.09,47.22c2.71,0,4.56,2.04,4.56,5.01v0.53h-6.77c0.15,1.15,1.07,2.11,2.62,2.11
+		c0.85,0,1.85-0.34,2.43-0.9l1.06,1.55c-0.91,0.83-2.34,1.26-3.75,1.26c-2.77,0-4.86-1.86-4.86-4.79
+		C273.37,49.35,275.32,47.22,278.09,47.22 M275.84,51.14h4.48c-0.06-0.87-0.66-2.02-2.24-2.02
+		C276.6,49.13,275.96,50.24,275.84,51.14"/>
+	<path class="st0" d="M289.85,55.61c-0.62,0.74-1.7,1.17-2.88,1.17c-1.45,0-3.17-0.98-3.17-3.02c0-2.13,1.72-2.9,3.17-2.9
+		c1.21,0,2.28,0.38,2.88,1.11v-1.26c0-0.92-0.79-1.53-2-1.53c-0.98,0-1.88,0.36-2.66,1.07l-0.91-1.6c1.11-0.98,2.54-1.43,3.98-1.43
+		c2.07,0,3.98,0.83,3.98,3.45v5.88h-2.39V55.61z M289.85,53.24c-0.4-0.53-1.15-0.79-1.92-0.79c-0.94,0-1.72,0.49-1.72,1.38
+		c0,0.85,0.77,1.34,1.72,1.34c0.77,0,1.53-0.26,1.92-0.79V53.24z"/>
+	<path class="st0" d="M295.09,54.29v-4.75h-1.51v-2.09h1.51v-2.49h2.41v2.49h1.85v2.09h-1.85v4.11c0,0.57,0.3,1,0.83,1
+		c0.36,0,0.7-0.13,0.83-0.26l0.51,1.81c-0.36,0.32-1,0.58-2,0.58C295.99,56.78,295.09,55.91,295.09,54.29"/>
+	<path class="st0" d="M307.15,51.01c0-1.26-0.66-1.66-1.7-1.66c-0.92,0-1.66,0.53-2.08,1.07v6.12h-2.39V43.98h2.39v4.65
+		c0.59-0.7,1.72-1.41,3.21-1.41c2,0,2.96,1.09,2.96,2.86v6.46h-2.39V51.01z"/>
+	<path class="st0" d="M312.17,53.69c0.66,0.62,2.02,1.24,3.15,1.24c1.04,0,1.53-0.36,1.53-0.92c0-0.64-0.79-0.87-1.83-1.07
+		c-1.56-0.3-3.58-0.68-3.58-2.85c0-1.53,1.34-2.86,3.73-2.86c1.54,0,2.77,0.53,3.67,1.24l-0.96,1.64c-0.55-0.58-1.58-1.06-2.7-1.06
+		c-0.87,0-1.43,0.32-1.43,0.85c0,0.56,0.72,0.77,1.73,0.98c1.56,0.3,3.66,0.72,3.66,2.98c0,1.66-1.41,2.92-3.94,2.92
+		c-1.58,0-3.11-0.53-4.07-1.41L312.17,53.69z"/>
+</g>
+</svg>

```

Comment:
This probably isn't super necessary as a practical thing but it might be at least a cool library to be aware of for dealing with SVGs. If you install `svgo`, a node.js tool for minimizing and optimizing SVG files, you could save at least 25% in file size. 

https://github.com/svg/svgo

Check it out and let me know if you need help running it on this file.

Reply:
Done! Looks like it reduced the file by about 27%. Thanks for the tip!